lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/QtiImport.java

```diff
@@ -1128,7 +1157,11 @@ boolean procshort(Node item) throws IOException{
 	    if (ident == null)
 		ident = ans.qident;
 
-	    out.println("<response_str ident=\""+rident+"\" rcardinality=\"Ordered\" rtiming=\"No\"><render_fib charset=\"ascii-us\" columns=\"20\" encoding=\"UTF_8\" fibtype=\"String\" prompt=\"Box\" rows=\"1\"></render_fib>");
+	    if (type.equalsIgnoreCase("Fill In the Blank")) {
+		    out.println("<response_str ident=\""+rident+"\" rcardinality=\"Ordered\" rtiming=\"No\"><render_fib charset=\"ascii-us\" columns=\"20\" encoding=\"UTF_8\" fibtype=\"String\" prompt=\"Box\" rows=\"1\"></render_fib>");
+	    } else if (type.equalsIgnoreCase("Numeric Response")) {
+		    out.println("<response_str ident=\""+rident+"\" rcardinality=\"Ordered\" rtiming=\"No\"><render_fin charset=\"ascii-us\" columns=\"20\" encoding=\"UTF_8\" fibtype=\"String\" prompt=\"Box\" rows=\"1\"></render_fin>");
```

Comment:
Same here. Spaces around the + signs, easier on the eye.

Reply:
Solved!